tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -104,6 +121,10 @@ sub rke2_server_setup {
         assert_script_run('curl -sfL https://get.rke2.io | sh -', timeout => 180);
     }
 
+    # Add kubectl command to $PATH environment varibable
+    my $rke2_bin_path = "export PATH=\$PATH:/opt/rke2/bin:/var/lib/rancher/rke2/bin";
+    assert_script_run("echo '$rke2_bin_path' > \$HOME/.bashrc; source \$HOME/.bashrc");
```

Comment:
>> or > here in 
```
echo '$rke2_bin_path' > \$HOME/.bashrc
```
?

Reply:
To avoid the existing content lost in this file, it should use '>>' instead of '>'. Thanks for pointing out.